packages/react/src/auth/useSignOut.ts

```diff
@@ -23,10 +24,14 @@ export const useSignOut = (opts?: { redirectOnSuccess?: boolean; redirectToPath?
 
     if (error) throw error;
 
-    if (redirectOnSuccess && data) {
-      const redirectUrl = new URL(redirectToPath ?? signInPath, window.location.origin);
-      window.location.assign(redirectUrl.toString());
-    }
+    // eslint-disable-next-line
+    useEffect(() => {
```

Comment:
This is needed because using something like `react-router-dom`s navigate means react re-rendering a different component while this one updates which results in
```Cannot update a component while rendering a different component warning```